### ISSUE TITLE
bug: disappearing part of query

### ISSUE THREAD

**Issue by Kamilcuk** [grafana/clickhouse-datasource#671]
Hi. I have a very old grafana plugin version. I have not found the similar issues. Because grafana plugin version is old, I am asking how to work around this bug?

**What happened**:

Part of query disappears. I do not understand the behavior.

This is the query:

```
SELECT  hostname AS h FROM ptpd.timekeeper
UNION SELECT (SELECT  hostname AS b FROM ptpd.ptpd)
```

But when I go into query inspector I only see:

```
SELECT ( SELECT hostname AS b FROM ptpd.ptpd) FROM ptpd.timekeeper UNION
```

I have tried various queries but half of them keeps dissapearing.

Additionally, I observe that it is impossible to query column named "offset", because it is replaced by uppercase.

For example: `select offset from table` is replaced by `select OFFSET from table`. There is no `OFFSET` column in the table, there is `offset`, lowercase.

**What you expected to happen**:

I write a query. Query is as it is.

**How to reproduce it (as minimally and precisely as possible)**:

1. Start creating a panel with clickhouse datasource
2. Write the query.
3. See it on query inspector
4. Compare - it's not the same!

**Screenshots**

![obraz](https://github.com/grafana/clickhouse-datasource/assets/15679217/eb63b274-95d6-442b-ac7a-ff8932155ff2)

![obraz](https://github.com/grafana/clickhouse-datasource/assets/15679217/c15efb6a-1f32-42fc-aa5a-cac6d6786bad)

**Anything else we need to know?**:

**Environment**:

- Grafana version: 9.2.4
- Plugin version: 1.1.2
- OS Grafana is installed on: Fedora 29
- User OS & Browser: Windows Chrome

**Comment by SpencerTorres** [comment]
That plugin version is indeed old... I don't think we will be able to support patches to that particular version.

Perhaps I could guide you in updating to a newer version of the plugin? Your version of Grafana supports version [3.x](https://github.com/grafana/clickhouse-datasource/releases/tag/v3.3.0) of the plugin.

If it's possible to upgrade your Grafana version to at least `9.5.0` you could use the new [4.x](https://github.com/grafana/clickhouse-datasource/releases) version of the plugin.

**Comment by SpencerTorres** [comment]
While your plugin version is outdated, we have fixed a similar issue in #684

Let me know if you need help upgrading plugin versions, for now I will close this issue.
